sections/annoyances.txt

```diff
@@ -75,6 +75,8 @@ blikk.hu###pe_confirm_optin_6
 blikk.hu##.leavingPopupLayer
 blikkruzs.blikk.hu###pe_confirm_optin_1
 blog.hu##[id*="cp_popup"]
+bme.hu###floating-logo-container
+bme.hu##^style:has-text(eu-cookie-withdraw-banner)
```

Comment:
This is a uBlock specific rule that cannot be in the common section, please move it to uBO section and add an AdGuard version as well